cypress/support/helpers/pageTypeSomeTestsNotRun.js

```diff
@@ -0,0 +1,4 @@
+const pageTypeSomeTestsNotRun = pageType =>
+  ['mostReadPage', 'errorPage404'].includes(pageType);
+
+export default pageTypeSomeTestsNotRun;
```

Comment:
We should find a better name for this function so that it's much clear I suggest splitting it up two functions eg.

```
const isErrorPage = pageType => pageType === 'errorPage404';

const isMostReadPage = pageType => pageType === 'mostReadPage';

const testRunSuccessfully = pageType => isErrorPage(pageType) || isMostReadPage(pageType);

export default testRunSuccessfully;
```

Reply:
I ended up removing this function (`pageTypeSomeTestsNotRun`) and opted for adding a specific if statement for where we do not want to run most read page (this is only in one place in this file: `cypress/integration/pages/testsForAllPages.js`).

Reply:
The tests are not being run for errorpages and most read with if statement when i tried it didnt work.Happy for you to see if that works.